spec/requests/sign_in_out_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe 'SignInOut', type: :request do
+  include ActionController::RespondWith
+  let(:user) { create(:user) }
+
+  context 'should get success status when sign_in' do
+    it 'get sign_in form' do
+      get '/#/sign_in'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'with right params' do
```

Comment:
it is not clear from the name what does this test check
please add appropriate verb to the test's name
it '<verb>s ...'